app/models/password_reset.rb

```diff
@@ -0,0 +1,36 @@
+class PasswordReset < ActiveRecord::Base
```

Comment:
Should this be a template instead of in `app/models`, like `lib/generators/clearance/install/templates/user.rb`?